vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -323,6 +323,19 @@ testAuditDNotPresent() {
   echo "$test:Finish"
 }
 
+testMessageOfTheDayComponentsAzureLinux() {
+  os_sku=$1
+  if [[ "$os_sku" == "CBLMariner" || "$os_sku" == "AzureLinux" ]]; then
```

Comment:
nit: can we use `isMarinerOrAzureLinux` this function instead?

Reply:
Will leave as is, that helper is in a completely different place, would need to source 600+ lines long script or refactor more.